components/script/dom/webidls/HTMLTableElement.webidl

```diff
@@ -6,7 +6,7 @@
 // https://www.whatwg.org/html/#htmltableelement
 interface HTMLTableElement : HTMLElement {
            attribute HTMLTableCaptionElement? caption;
-  HTMLElement createCaption();
```

Comment:
It lets us get rid of the HTMLElementCasts, and it doesn't really matter what type you declare as being returned in the IDL, what matters is the properties/methods on the type actually returned.